go

```diff
@@ -34,7 +34,7 @@ def_command :update_gems, 'Update Ruby gems' do |gems = []|
 end
 
 def_command :test, 'Execute automated tests' do |args = []|
-  exec_cmd "bundle exec rake test #{args.join ' '}"
```

Comment:
Ah... Probably because the Rakefile doesn't have an Rspec rule... Well if this works for both the Rspec stuff and the other tests, good by me, but you probably still want to run `bundle exec rspec`.